server/lib/puppet-languageserver/puppet_parser_helper.rb

```diff
@@ -1,15 +1,28 @@
 module PuppetLanguageServer
   module PuppetParserHelper
-    def self.remove_char_at(content, line_offsets, line_num, char_num)
+    def self.remove_chars_starting_at(content, line_offsets, line_num, char_num, move_offset)
```

Comment:
Is `move_offset` the best name for the parameter here?  Is this the number of chars to remove?